README.md

```diff
@@ -1,177 +1,164 @@
+1. The next step is to create a persistent volume claim. Choose the storage class you need &mdash; either gold, silver, or bronze. Add your name as `fp-predict-plus-pvc`, select single user access, and assign the storage size as 20 GB. 
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/create-pvc.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/create-pvc.png)
 
-After the persistent volume claim (PVC) is created, it needs to be bound with the persistent volume (PV) which was created in earlier step. We should see the status as `Bound` per below after couple of minutes.
+1. After you create the persistent volume claim (PVC), you need to bind it with the persistent volume (PV) which you created in tye earlier step. You should see the status as `Bound` per below after couple of minutes.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/pvc-bound.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/pvc-bound.png)
 
-### Install the operand (Instance) of FPPredict Plus
+## Install the instance of FPPredict Plus
 
 Click on Installed operators under `Operators` and click on FP Predict Plus Operator to get the options like Overview, YAML, Subscription, Events, FP-Predict-Plus. Click on YAML and update the name per below under persistent volume with `useExisting as false`, name of persistent volume claims, routerCanonicalHostname would be the web console URL and hit `Save`. routerCanonicalHostname would start with the cluster name, cluster id till appdomain.cloud. The initial part in the URL - `https://console-openshift-console` should be removed while updating routerCanonicalHostname.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/yaml-changes.png)
+![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/yaml-changes.png) <!--EM: Is this the wrong image? It doesnt' seem to go with teh text above-->
 
 The next step is to proceed towards FP-Predict-Plus option and click on Create FPpredictplus instance. Edit the YAML file and give a name for the instance and click `Save`.
 
 ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/operand.png)
 
-### Launch the operand (Instance) of FPPredict Plus
+### Launch the instance of FPPredict Plus
 
-We are all set to launch the instance. `How do we do it?` We need to click on Networking and select Routes and then click on the URL which is under the location to launch the instance. 
+Now you are set to launch the instance. To do so, click on Networking and select Routes and then click on the URL which is under the location to launch the instance. 
 
 ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/route.png)
+<!--EM: Is this the right image?-->
 
```

Comment:
The image is right, the complete URL under location is not shown intentionally but the users will get an idea where to access the URL.